src/contracts/pods/EigenPod.sol

```diff
@@ -199,102 +188,93 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
-        require(
-            oracleTimestamp + VERIFY_BALANCE_UPDATE_WINDOW_SECONDS >= block.timestamp,
-            "EigenPod.verifyBalanceUpdate: specified timestamp is too far in past"
-        );
-
-        uint64 validatorBalance = BeaconChainProofs.getBalanceFromBalanceRoot(validatorIndex, balanceUpdateProof.balanceRoot);
-
-        /** 
-        * Reference: 
-        * uint64 validatorWithdrawableEpoch = Endian.fromLittleEndianUint64(validatorFields[BeaconChainProofs.VALIDATOR_WITHDRAWABLE_EPOCH_INDEX]);
-        * uint64 oracleEpoch = _computeSlotAtTimestamp(oracleTimestamp)) / BeaconChainProofs.SLOTS_PER_EPOCH;
-        * require(validatorWithdrawableEpoch > oracleEpoch)
-        * checks that a balance update can only be made before the validator is withdrawable.  If this is not checked
-        * anyone can prove the withdrawn validator's balance as 0 before the validator is able to prove their full withdrawal
-        */
-        if (
-            Endian.fromLittleEndianUint64(validatorFields[BeaconChainProofs.VALIDATOR_WITHDRAWABLE_EPOCH_INDEX]) <=
-            (_computeSlotAtTimestamp(oracleTimestamp)) / BeaconChainProofs.SLOTS_PER_EPOCH
-        ) {
-            require(validatorBalance > 0, "EigenPod.verifyBalanceUpdate: validator is withdrawable but has not withdrawn");
-        }
         
-        bytes32 validatorPubkeyHash = validatorFields[BeaconChainProofs.VALIDATOR_PUBKEY_INDEX];
-
+        uint64 validatorBalance = balanceUpdateProof.balanceRoot.getBalanceAtIndex(validatorIndex);
+        bytes32 validatorPubkeyHash = validatorFields.getPubkeyHash();
         ValidatorInfo memory validatorInfo = _validatorPubkeyHashToInfo[validatorPubkeyHash];
 
-        // verify that the validator has been proven to have its withdrawal credentials pointed to this EigenPod, and has not yet been proven to be exited
-        require(validatorInfo.status == VALIDATOR_STATUS.ACTIVE, "EigenPod.verifyBalanceUpdate: Validator not active");
 
-        // check that the balance update is being made strictly after the previous balance update
+        // Verify balance update timing:
+
+        // 1. Balance updates should only be performed on "ACTIVE" validators
+        require(
+            validatorInfo.status == VALIDATOR_STATUS.ACTIVE, 
+            "EigenPod.verifyBalanceUpdate: Validator not active"
+        );
+
+        // 2. Balance updates should be more recent than the most recent update
         require(
             validatorInfo.mostRecentBalanceUpdateTimestamp < oracleTimestamp,
             "EigenPod.verifyBalanceUpdate: Validators balance has already been updated for this timestamp"
         );
 
-        {
-            // verify ETH validator proof
-            bytes32 latestBlockRoot = eigenPodManager.getBlockRootAtTimestamp(oracleTimestamp);
+        // 3. Balance updates should not be "stale" (older than 4.5 hrs)
```

Comment:
grouping this checks together is lit, can we make this comment a bit more clear lik "balance updates must be made within VERIFY_BALANCE_UPDATE_WINDOW_SECONDS of the oracleTimeStamp" or smthg

Reply:
The comment you linked specifically references the value backed by that constant, and the code itself uses the constant -- do you think this needs additional clarity?

Reply:
I could update the comment to mention the oracle timestamp - would that be better?

Reply:
IMO it's OK, but I do think referencing the constant rather than the value tends to make more sense, just to ensure that the comment remains accurate, e.g. if the definition of the constant is ever changed.

Reply:
aight -- that's a good point. pushed a fix